Jenkinsfile

```diff
@@ -10,6 +10,7 @@ pipeline {
     options {
         timestamps() 
         skipStagesAfterUnstable()
+        timeout(time: 1, unit: 'HOURS')
```

Comment:
I had changed the timeout on the master branch to 2 hours because 1 hour was cutting it close.

Reply:
updated timeout to 2 hours. Thanks!